java/pom.xml

```diff
@@ -140,6 +140,16 @@
       <artifactId>spark-cassandra-connector_${scala.binary.version}</artifactId>
       <version>3.1.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.mongodb</groupId>
+      <artifactId>mongo-java-driver</artifactId>
+      <version>3.12.12</version>
+    </dependency>
+    <dependency>
+      <groupId>org.mongodb.spark</groupId>
+      <artifactId>mongo-spark-connector_2.12</artifactId>
```

Comment:
Done.

Reply:
@saumyasinha-google Just wanted to understand that, do we actually need to make changes in pom.xml file? As far as I remember, for all the previously built mongo templates, we are explicitly providing the jar files. 

Please do let me know, if I am missing out something over here.

Reply:
@hhasija It's not needed, we can provide the connector and driver in jar files. 
Since spark connector for cassandra and a few other templates was added as dependencies in pom.xml hence added dependencies for mongo as well.
Should they be provided in jar files only?

Reply:
Yes, let us go ahead by providing them in jar files only. Because, going ahead whenever version mis match occurs, it requires us to change it in pom.xml file otherwise, we could just replace the jar files and get the things done.

Reply:
Done.